SQLite/Foundation.swift

```diff
@@ -47,7 +47,7 @@ extension NSDate : Value {
     }
 
     public class func fromDatatypeValue(stringValue: String) -> NSDate {
-        return dateFormatter.dateFromString(stringValue)!
+        return dateFormatter.dateFromString(stringValue) ?? NSDate()
```

Comment:
The problem with this is that it makes it seem like the database is actually storing a string that represents the current date, which can be misleading and confusing to users.